library/core/src/task/wake.rs

```diff
@@ -260,6 +276,14 @@ impl Waker {
     pub unsafe fn from_raw(waker: RawWaker) -> Waker {
         Waker { waker }
     }
+
+    /// Get the reference to the underlying [`RawWaker`].
+    #[inline]
+    #[must_use]
+    #[unstable(feature = "waker_getters", issue = "87021")]
+    pub fn as_raw_waker(&self) -> &RawWaker {
```

Comment:
```suggestion
    pub fn as_raw(&self) -> &RawWaker {
```

For consistency with `from_raw`